pypeapp/lib/log.py

```diff
@@ -51,7 +51,10 @@
 
 system_name, pc_name = platform.uname()[:2]
 host_name = socket.gethostname()
-host_id = socket.gethostbyname(host_name)
+try:
+    ip = socket.gethostbyname(host_name)
+except socket.gaierror:
+    ip = "127.0.0.1"
```

Comment:
Just note: I think host IP was added by me for more information in log records but I don't think It's required and may be removed....